### PR TITLE
Add support for redis 5 zpopmin and zpopmax

### PR DESCRIPTION
**Pull request by maximebedard**
Description: Add support for redis 5 [zpopmin](https://redis.io/commands/zpopmin) and [zpopmax](https://redis.io/commands/zpopmax). They behave as a simple comment as the first argument is the key of the sorted set. 
Risk Level: low
Testing: N/A
Docs Changes: Updated list of supported commands
Release Notes: Added new commands to release note.

### CODE REVIEW

**Comment by FAYiEKcbD0XFqF2QK2E4viAHg8rMm2VbjYKdjTg** [comment]
LGTM

File: docs/root/intro/version_history.rst [hunk 1]
```diff
@@ -4,6 +4,7 @@ Version history
 1.11.0 (Pending)
 ================
 * upstream: added :ref:`upstream_cx_pool_overflow <config_cluster_manager_cluster_stats>` for the connection pool circuit breaker.
+* redis: add support for zpopmin and zpopmax commands.
```
Comment (mattklein123) [inline docs/root/intro/version_history.rst]: Sorry just missed this. Can you alpha sort? Sorry and than you.

Comment (mattklein123) [reply docs/root/intro/version_history.rst]: Sorry I actually meant alpha sort this entry in the version history, but thanks for the other one also. :)

Might as well merge master again also.

/wait

**Comment by maximebedard** [comment]
Thank you! 🙏

**Pull request merged**